src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
```

Comment:
Although the task requirement is met, the promise1 will only be resolved once the logo is clicked. This might not be the desired behavior if multiple clicks on the logo are expected to trigger different actions. Consider moving the event listener outside of the promise.